palindrome.c

```diff
@@ -0,0 +1,32 @@
+#include <stdio.h>
```

Comment:
Great job! The only thing is you forgot to handle capitals (so that "Hannah" would be considered a palindrome), but good job! 👍